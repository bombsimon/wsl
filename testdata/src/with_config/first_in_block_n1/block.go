package testpkg

func Fn(_ int) {}

func fn1() {
	one := 1
	two := 2
	three := 3

	a := 1
	if true {
		Fn(a)
	}

	b := 1
	if true { // want `missing whitespace above this line \(no shared variables above if\)`
		Fn(one)

		if false {
			Fn(two)

			if true {
				Fn(three)

				if true {
					Fn(b)
				}
			}
		}
	}
}
