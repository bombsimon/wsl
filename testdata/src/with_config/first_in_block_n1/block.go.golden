package testpkg

func Fn(_ int) {}

func fn1() {
	one := 1
	two := 2
	three := 3

	a := 1
	if true {
		Fn(a)
	}

	b := 1

	if true { // want `missing whitespace decreases readability \(if\)`
		Fn(one)

		if false {
			Fn(two)

			if true {
				Fn(three)

				if true {
					Fn(b)
				}
			}
		}
	}
}

